BambeeHR/pollen
# feat(PRIVA-1355): automate releases when version changes

## Issue(s)

https://bambee.atlassian.net/browse/PRIVA-1355

## Description

See attached Jira issue for AC.

**NOTE:** The Platform team plans to formalize this into the `ops-platform` over the coming weeks which is why I'm using an off the shelf action (that hasn't been updated in 2 years)

## Testing

- Change the value of the `version` in the `package.json` file.
- Push to `master`
- See repository tagged with new version
- See package published to GitHub package registry

## Risk Assessment

low
Change the scope back to `@bambee` and continue to manually publish if we run into problems with this
